pandas/core/reshape/merge.py

```diff
@@ -47,7 +47,7 @@
 def concat_wrap():
 
     def wrapper(*args, **kwargs):
-        warnings.warn("pandas.tools.merge.concat is deprecated. "
```

Comment:
hmm, this was original ``pandas.tools.merge.concat``; I will put a deprecation there then.